README.md

```diff
@@ -1,23 +1,33 @@
-# React to GitHub Pages
+# Deploy React to GitHub Pages 
 
-GitHub action to build and deploy React to GitHub Pages.  This uses the new GitHub Actions method as the source.  In the repository settings, go to Pages.  Under Source, select GitHub Actions.  No further configuration is needed.  
+`bitovi/github-actions-react-to-github-pages` builds and deploys a React application to GitHub Pages.
 
-## Customizing
+This action uses the [new GitHub Actions](https://www.bitovi.com/blog/deploy-your-react-project-to-github-pages-with-github-actions#DeployyourReactprojecttoGitHubPageswithGitHubActions-What%E2%80%99sNewwithGitHubPages) publishing method which allows you to create an artifact that contains the result of the build and serves the files in the artifact on the Pages site. There’s no need to check files back into your repository, keeping it nice and clean.
 
-### Inputs
 
-The following inputs can be used as `step.with` keys
+## You are here
+This action deploys React/Javascript to Github Pages.  The build process should create static files and put them into a build direcory that will be moved into your Pages hosting location.  
 
-| Name             | Type    | Description                        |
-|------------------|---------|------------------------------------|
-| `checkout`          | T/F  | Set to `false` if the code is already checked out (Default is `true`) (Optional) |
-| `path` | String | Path of output files, Default is `dist` (Optional)|
-| `build_command` | String | Specifies the command to run after `npm ci` for the build, Default is `npm run build` (Optional)|
+If you would like to deploy a backend app/service, check out our other actions:
+| Action | Purpose |
+| ------ | ------- |
+| [Deploy Docker to EC2](https://github.com/bitovi/github-actions-deploy-docker-to-ec2) | Deploys a repo with a Dockerized application to a virtual machine (EC2) on AWS |
```

Comment:
Should we add 
[Deploy static site to AWS (S3/CDN/R53)[https://github.com/marketplace/actions/deploy-static-site-to-aws-s3-cdn-r53] | Deploys a static site to .....